airbyte-integrations/connector-templates/source-python-http-api/setup.py.hbs

```diff
@@ -20,11 +20,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-protocol",
-    "base-python",
+    "airbyte-cdk",
+    "requests",
```

Comment:
is this needed? This should be pulled in by cdk

Reply:
good point. not sure why my IDE was complaining

are python dependencies transitive?

Reply:
yes they are. removing this for now